bundles/org.openhab.binding.volumio/src/main/java/org/openhab/binding/volumio/internal/VolumioHandler.java

```diff
@@ -274,10 +324,18 @@ public void initialize() {
             logger.debug("Trying to connect to Volumio on {}://{}:{}", protocol, hostname, port);
             try {
                 volumio = new VolumioService(protocol, hostname, port, timeout);
+                VolumioService volumioLocal = volumio;
+                if (volumioLocal == null) {
+                    if (ThingStatus.ONLINE.equals(getThing().getStatus())) {
+                        updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR,
+                                "Error during volumio service initialization.");
+                    }
+                    return;
+                }
```

Comment:
It can never be null if you write it like this:

```suggestion
                VolumioService volumioLocal = new VolumioService(protocol, hostname, port, timeout);
                volumio = volumioLocal;
```

Reply:
solved